rfc/.template.md

```diff
@@ -1,90 +0,0 @@
-# _<short_summary_of_proposal>_
```

Comment:
Was this deleted intentionally?

Reply:
Do we need to keep this file?

Reply:
Yes, its purpose is to serve as a template for future RFC authors.

Reply:
Added this file back.